Types/Id.php

```diff
@@ -59,7 +60,29 @@ public static function create(array $tokens) {
      * @SuppressWarnings("PHPMD.StaticAccess")
      */
     public static function alias(array $tokens) {
+        $column     = self::column($tokens, new MetaData());
         $tableAlias = Table::alias($tokens);
-        return empty($tableAlias) ? 'id' : $tableAlias . '.id';
+
+        return empty($tableAlias) ? $column : $tableAlias . '.' . $column;
+    }
+
+    /**
+     * returns the column of the id
+     *
+     * @param  array    $tokens
+     * @param  MetaData $metaData
+     * @return string
+     *
+     * @SuppressWarnings("PHPMD.StaticAccess")
+     */
+    public static function column(array $tokens, MetaData $metaData) {
```

Comment:
das versteh ich nich. wie kann denn ne id ne column haben, wenns en integer is? hat nich eher ne column ne id und wär das dann nich Column.create: Id -> Column